render/renderer.go

```diff
@@ -39,10 +39,29 @@ func SixelSupported() bool {
 	if err != nil {
 		return false
 	}
-	if !bytes.HasPrefix(b[:n], []byte("\x1b[?63;")) {
+	var supportedTerminals = []string{
+		"\x1b[?62;", // VT240
+		"\x1b[?63;", // wsltty
+		"\x1b[?64;", // mintty
+		"\x1b[?65;", // RLogin
+	}
+	supported := false
+	for _, supportedTerminal := range supportedTerminals {
+		if bytes.HasPrefix(b[:n], []byte(supportedTerminal)) {
+			supported = true
+			break
+		}
+	}
+	if !supported {
 		return false
 	}
-	for _, t := range bytes.Split(b[4:n], []byte(";")) {
+
+	sb := b[6:n]
```

Comment:
Could you share some links to understand these magic numbers, such as `6` or `4`?
I guess it's something according to the response format of DA, though.

and I found your tweet when I googled... 🙇 
* https://twitter.com/ttdoda/status/1174261417366736896
* https://stackoverflow.com/questions/18379477/how-to-interpret-response-from-vt-100-vt-102-da-request
* https://www.vt100.net/docs/vt100-ug/chapter3.html#DA

If there is an official link as a reference I can learn these stuff, it would be super helpful.

Reply:
https://invisible-island.net/xterm/ctlseqs/ctlseqs.html

![image](https://user-images.githubusercontent.com/10111/114717383-f4449c80-9d6f-11eb-929b-20b692679c01.png)

If sending `CSI c` (Ps is omitted), terminals send back them attributes. 

![image](https://user-images.githubusercontent.com/10111/114717734-4eddf880-9d70-11eb-9ba0-20cfec2fda0e.png)

As you can see, 62 is VT220. Following attributes are features that the terminal has. The magic number `6` is offset value for skip decTerminalID. And `4` is Sixel graphics.